### ISSUE TITLE
Do you need mongo db running?

### ISSUE THREAD

**Issue by spences10** [notrab/graphql-server-tutorial#2]
I copied your code from the initial commit 19cf7f1fc2819bd0fd940da26b1071199e80a50d as I couldn't see where I went wrong following along but I get an unhandled error warning with the code from the initial commit, sample output here:

```shell
λ npm run dev

> graphql-server@1.0.0 dev C:\Users\spenc\gitrepos\graphql-server
> node src

(node:5508) DeprecationWarning: `open()` is deprecated in mongoose >= 4.11.0, use `openUri()` instead, or set the `useMongoClient` option if using `connect()` or `createConnection()`. See http://mongoosejs.com/docs/connections.html#use-mongo-client
> Ready on PORT 5000

events.js:182
      throw er; // Unhandled 'error' event
      ^
MongoError: failed to connect to server [undefined:27017] on first connect [MongoError: getaddrinfo ENOTFOUND undefined undefined:27017]
```

Seems that mongoose wants some config info but you haven't covered it in the first video, what should I do? 

I can't seem to get to the point where in the video you use graphiql, I don't really want to keep going through the videos until I get the something running.

**Comment by spences10** [comment]
Just for info, here is the error I get trying to follow along:

```shell
λ npm run dev

> graphql-server@1.0.0 dev C:\Users\spenc\gitrepos\graphql-server
> nodemon src

[nodemon] 1.11.0
[nodemon] to restart at any time, enter `rs`
[nodemon] watching: *.*
[nodemon] starting `node src`
C:\Users\spenc\gitrepos\graphql-server\node_modules\graphql\jsutils\invariant.js:19
    throw new Error(message);
    ^

Error: Event.date field type must be Output Type but got: undefined.
```

**Comment by notrab** [comment]
Hey @spences10 

Thanks for pointing this out. I actually cover this VERY briefly at [1:19](https://youtu.be/1z7cz1A5jzU?t=1m19s).

You'll need to type `export MONGODB_URI=mongodb://localhost/<nameOfDbHere>` in the terminal before running the server.

I made the mistake of leaving the video on top of the terminal in this video.

I made the assumption those following have experience with Node, Mongo and at least have it running. Apologies for that - I'll know for the future.
